doc/posts/2023/2023_08_21_vara_week_12_13.rst

```diff
@@ -8,36 +8,57 @@ Finalized experiments using both datasets: Week 12 & Week13
+Here's a similar visualization of the inference on the next best performing model, B12-CC. 
+
+.. image:: /doc/_static/vqvae-monai-B12-CC.png
+   :alt: VQVAE-Monai-B12-CC reconstructions & originals showing equally spaced 5 slices for 2 different test samples
+   :width: 800
+
+This shows that our training not only converged quickly but also improved visually. Here's a comparison of our current best performing model i.e., VQVAE-Monai-B12-both & the previous one on NFBS i.e., VQVAE-Monai-B5-NFBS. The test reconstruction loss is 0.0013 & 0.0015 respectively. 
 
 .. image:: /doc/_static/vqvae-reconstructions-comparison.png
    :alt: VQVAE reconstruction comparison for B12-both & B5-NFBS
    :width: 800
 
-I also carried Diffusion Model training for the bets performing B12-both model for 300 & 500 diffusion steps and the training curve obtained is as follows-
+I also carried out Diffusion Model training on the best performing VQVAE-Monai-B12-both model for 300 & 500 diffusion steps and the training curves obtained are as follows-
 
 .. image:: /doc/_static/dm3d-monai-training-curves.png
    :alt: Diffusion Model training plots for 300 & 500 diffusion steps
    :width: 800
 
-These curves seemed to converge pretty quickly but the sampling outputs in the generation pipeline are still pure noise.
+These curves seemed to converge pretty quickly but the sampling outputs in the generation pipeline are still pure noise as shown below-
+
+.. image:: /doc/_static/dm3d-monai-B8-DM500.png
+   :alt: Diffusion Model training plots for 300 & 500 diffusion steps
+   :width: 800
+
+
+The best performance of KL encoders/VAEs/VQVAEs have been shown to deliver blurry reconstructions on Medical datasets. Despite using a pretty less complex VQVAE model, with only ``num_res_channels=(32, 64)``, we consistently achieved improved reconstruction results with every experiment. From capturing only the brain outer structure through VQAVE-B5-NFBS model to able to capture the volumetric details of the inner microstrcuture of the brain through VQVAE-Monai-B10-both model, we've stretched teh capabilties of the VQVAE model. 
+
+For future work we should look into two things - debugging Diffusion Model, scaling VQVAE model. 
+
+As a first priority, we could analyze the reason for pure noise output in DM3D generations, this would help us rule out any implementation erros(although I already did that, second pair of eyes would be helpful). Because MONAI's work shows that even without a stable convergence on their diffusion model they achieved visualizations resembling a brain. Where as our model despite achieving a stable looking training curve, our generations are no close to being convincing.
+
+As a second step, we could also try sclaing up both VQVAE as well as Diffusion Model interms of complexity, such as increasing intermediate channel dimensions from 64 to 128 or 256 as done in MONAI's 3D LDM. This hopefully may help us achieve the state-of-art on NFBS & CC359 datasets.
```

Comment:
sclaing -> scaling
Please copy your text to word or some typo checking software to catch these.

Reply:
Also check the spelling errors that github caught